recipe/meta.yaml

```
@@ -57,6 +57,8 @@ outputs:
         - libcurand        # [(cuda_compiler_version or "").startswith("12")]
         - libcusolver      # [(cuda_compiler_version or "").startswith("12")]
         - libcusparse      # [(cuda_compiler_version or "").startswith("12")]
+        # see discussions/references in conda-forge/cupy-feedstock#247
+        - cuda-version
```

Do we not want to constrain this? Like:
```suggestion
        - cuda-version >=12.0,<13.0a0  # [(cuda_compiler_version or "").startswith("12")]
        - cuda-version >=11.0,<12.0a0  # [(cuda_compiler_version or "").startswith("11")]
```

---

We could do that; I didn’t because I found it’s enough to leave it unconstrained (and `cupy-core`’s constraints would kick in) yet allow the solver to properly consider it.